src/plugin/index.js

```diff
@@ -33,8 +34,9 @@ function Jarvis(options = {}) {
 
 Jarvis.prototype.apply = function(compiler) {
   const { name, version, author: makers } = pkg;
-  this.reports.project = { name, version, makers };
+  const normalizedAuthor = parseAuthor(makers);
```

Comment:
This won't work because `parseAuthor` is a `const` defined after `normalizedAuthor`; `const`s don't get hoisted like `var`s. You can make `parseAuthor` a function + leave it where it is, but I'd rather see it defined before(`const` or `function`)  `Jarvis.prototype.apply` to lessen the complexity + improve readability/modularity.

![screen shot 2018-01-18 at 7 02 12 pm](https://user-images.githubusercontent.com/5226549/35127916-22abc4be-fc82-11e7-933a-a16642e2b988.png)